src/components/structures/MatrixChat.tsx

```diff
@@ -881,7 +892,7 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
                 });
                 break;
             case "client_started":
-                this.onClientStarted();
+                this.onClientStarted().then();
```

Comment:
driveby comment: what's with the empty `then()` ?

Reply:
Just as a way of indicating that we're deliberately not waiting for an async function, but I guess it didn't work. :) I forget if we decided on a preferred way of doing this - `(void)this.onClientStarted();` maybe? Or just a comment?

Reply:
A comment to explain why it's needed is welcomed :)